tests/test_dht_node.py

```diff
@@ -217,9 +221,9 @@ def test_dht_node():
         jaccard_denominator += k_nearest
 
     accuracy = accuracy_numerator / accuracy_denominator
-    print("Top-1 accuracy:", accuracy)  # should be 98-100%
+    logger.info(f"Top-1 accuracy: {accuracy}")  # should be 98-100%
```

Comment:
For these two, even logger.debug is sufficient (we’ll know if they are too low when the assertion below fails)